src/components/CommentList.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../api";
+import { CommentCard } from "./CommentCard";
+
+export const CommentList = ({review_id}) => {
+
+    const [listOfComments, setListOfComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        setIsLoading(true)
+        getComments(review_id).then((comments) => {
+            setListOfComments(comments)
+            setIsLoading(false)
+        });
+    }, []);
+
+    return isLoading ? (
+        <p> Loading comments... </p>
+    ) :
+    listOfComments.length === 0 ?  (<div><p> No comments yet </p>
```

Comment:
Really like the way you're checking the comments length and displaying a message if they don't exist 👍